src/nyc_trees/apps/home/urls.py

```diff
@@ -17,7 +17,9 @@
     url(r'^jobs/(?P<job_id>\d+)/$', r.retrieve_job_status,
         name='retrieve_job_status'),
 
-    url(r'^training/$', r.training_list_page),
+    url(r'^training/$', r.training_list_page, name="training_list_page"),
+
+    url(r'^training/groups_to_follow/$', r.groups_to_follow),
```

Comment:
It appends `/` for groups, events, etc. pages but not the `training/groups_to_follow` page. I can test again later to verify.